### ISSUE TITLE
Add support for Slicer Segmentation Objects as masks

### ISSUE THREAD

**Issue by muratmaga** [SlicerMorph/SlicerANTs#6]
Currently masked registration requires the masks to be labelmaps, which makes sense. However, since this version of ANTsRegistration is specific to Slicer, it would be nice to use the segmentations as masks as well. 

Would it be possible to add that as well (i.e., support both segmentations and labelmaps). 

if it is extra burden complexity to implement that, at least add a tooltip that says "Make sure you have exported (or loaded) your segmentation as a labelmap".

**Comment by thewtex** [comment]
@muratmaga good idea, yes we will add this. Internally, we can convert the segmentation to a labelmap when it is passed to ANTs.
